cmd/tableprinters/switch.go

```diff
@@ -59,8 +59,13 @@ func (t *TablePrinter) SwitchTable(data []*models.V1SwitchResponse, wide bool) (
 			mode = "operational"
 		}
 
+		os := ""
+		if s.Os != nil {
+			os = s.Os.Vendor + "/" + s.Os.Version
```

Comment:
Can look iffy when things like version could not be figured out?

Reply:
done